ui/ozone/platform/x11/x11_display_fetcher_ozone.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef UI_OZONE_PLATFORM_X11_X11_DISPLAY_FETCHER_OZONE_H_
+#define UI_OZONE_PLATFORM_X11_X11_DISPLAY_FETCHER_OZONE_H_
+
+#include <stdint.h>
+
+#include "ui/display/types/native_display_delegate.h"
+#include "ui/events/platform/platform_event_dispatcher.h"
+
+typedef unsigned long XID;
+typedef XID Window;
+typedef struct _XDisplay Display;
+
+namespace display {
+class Display;
+class DisplayMode;
+}  // namespace display
+
+namespace ui {
+
+// X11DisplayFetcherOzone talks to xrandr.
+class X11DisplayFetcherOzone : public ui::PlatformEventDispatcher {
+ public:
+  class Observer {
```

Comment:
it sounds more like a Delegate rather than Observer.

Reply:
Done.